CodeSniffer/Standards/Generic/Sniffs/PHP/DisallowAlternativePHPTagsSniff.php

```diff
@@ -0,0 +1,253 @@
+<?php
+/**
+ * Generic_Sniffs_PHP_DisallowAlternativePHPTagsSniff.
+ *
+ * PHP version 5
+ *
+ * @category  PHP
+ * @package   PHP_CodeSniffer
+ * @author    Greg Sherwood <gsherwood@squiz.net>
+ * @author    Marc McIntyre <mmcintyre@squiz.net>
+ * @copyright 2006-2014 Squiz Pty Ltd (ABN 77 084 670 600)
+ * @license   https://github.com/squizlabs/PHP_CodeSniffer/blob/master/licence.txt BSD Licence
+ * @link      http://pear.php.net/package/PHP_CodeSniffer
+ */
+
+/**
+ * Generic_Sniffs_PHP_DisallowAlternativePHPTagsSniff.
+ *
+ * Verifies that no alternative PHP tags are used.
+ *
+ * If alternative PHP open tags are found, this sniff can fix both the open and close tags.
+ *
+ * @category  PHP
+ * @package   PHP_CodeSniffer
+ * @author    Greg Sherwood <gsherwood@squiz.net>
+ * @author    Marc McIntyre <mmcintyre@squiz.net>
+ * @copyright 2006-2014 Squiz Pty Ltd (ABN 77 084 670 600)
+ * @license   https://github.com/squizlabs/PHP_CodeSniffer/blob/master/licence.txt BSD Licence
+ * @version   Release: @package_version@
+ * @link      http://pear.php.net/package/PHP_CodeSniffer
+ */
+class Generic_Sniffs_PHP_DisallowAlternativePHPTagsSniff implements PHP_CodeSniffer_Sniff
+{
+
+
+    /**
+     * Whether ASP tags are enabled or not.
+     *
+     * @var bool
+     */
+    private $_asp_tags = false;
+
+
+    /**
+     * Returns an array of tokens this test wants to listen for.
+     *
+     * @return array
+     */
+    public function register()
+    {
+        if (version_compare(PHP_VERSION, '7.0.0alpha1', '<') === true) {
```

Comment:
There is at least [one other place](https://github.com/squizlabs/PHP_CodeSniffer/blob/master/CodeSniffer/Standards/Generic/Tests/PHP/DisallowShortOpenTagUnitTest.php#L51) where versions like this are checked against pre-release version nrs, so I figured I'd better follow suit.

All the same, IMHO, it would be better to use the `PHP_VERSION_ID` constant as that way pre-release versions would be automatically included.
I looked into using that, but that constant was only [introduced in PHP 5.2.7](http://php.net/manual/en/reserved.constants.php#reserved.constants.core), while PHPCS has a minimum PHP requirement of 5.1.2.

It would be easy enough to define the constant in the PHPCS loader in case it's not available and I'd be happy to send in a PR to that end, but I figured that ought to be a separate issue/PR.